examples/js/loaders/ColladaLoader.js

```diff
@@ -1499,6 +1552,16 @@ THREE.ColladaLoader.prototype = {
 				}
 
 			}
+			
+			if ( extra !== undefined && extra.technique !== undefined && isEmpty( extra.technique ) === false ) {
+                
+                		if ( extra.technique.double_sided !== undefined && extra.technique.double_sided === 1) {
```

Comment:
One little thing. Please add a space after `1`. After that, the code style looks is perfect 😇